packages/web/src/application/setup.jest.ts

```diff
@@ -1,8 +1,14 @@
 import application from '@application';
+import crypto from 'crypto';
+import fetch from 'node-fetch';
 
 import '../modules';
 
-beforeAll(() => application.init());
+// required for ZoomSdk
+Object.defineProperty(global.self, 'crypto', {
+  value: crypto.webcrypto,
+});
+
+Object.defineProperty(global.self, 'fetch', { value: fetch });
 
-console.warn('See jest.config. - TLDR; upgrade to jest 28.0.6 when possible');
```

Comment:
Leave this warning, I added it with purpose.

Reply:
Isn't it upgraded?
`"ts-jest": "^28.0.8",`
`"@types/jest": "^28.1.8",`

Reply:
You’re referring to ts-jest which uses jest under the hood. The new version of jest is needed.

Reply:
Should I upgrade? This could fix the `waitFor` thing.

Reply:
You cannot so easily because jest comes with react-scripts in CRA setup.

I'll take a look at waitFor issue today